extensions/README.md

```diff
@@ -102,3 +102,55 @@ To test extensions for a Simple site in Calypso, sandbox the simple site URL (ex
 If you need to modify the build process, bear in mind that config files are also
 synced to WordPress.com via Fusion. Consult with a Jetpack crew member to ensure
 you test the new build in both environments.
+
+## Good to know when developing Gutenberg extensions
+
+### Don't worry about dependencies
+
+The build takes care of core dependencies for both editor and view -scripts. React, lodash and `@wordpress/*` [dependencies](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/scripts.md) are externalised and automatically enqueued in PHP for your extension.
+
+Extensions _always_ get [Gutenberg's polyfill scripts](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/scripts.md#polyfill-scripts) enqueued so you can safely use methods not supported by older browsers such as IE11.
+
+### Jetpack sidebar
+
+Jetpack sidebar is registered on all Gutenberg editor pages and filled using [Slot Fill](https://github.com/WordPress/gutenberg/tree/master/packages/components/src/slot-fill#readme).
+
+Just render to `JetpackPluginSidebar` from anywhere in the code:
+
+```jsx
+import JetpackPluginSidebar from '../../shared/jetpack-plugin-sidebar';
+
+<JetpackPluginSidebar>
+	<PanelBody title={ __( 'My sidebar section', 'jetpack' ) }>
+		<p>Jetpack is Bestpack!</p>
+	</PanelBody>
+</JetpackPluginSidebar>
+```
+
+Remember to be mindful of posts, pages, custom post types and re-usable blocks.
+
+### i18n
+As of 04/2019 `wp.i18n` [doesn't support React elements in strings](https://github.com/WordPress/gutenberg/issues/9846). You will have to structure your copy so that links and other HTML can be translated separately.
+
+Not possible:
+
+> Still confused? Check out <a>documentation</a> for more!
+
+Possible:
+
+> Still confused? <a>Check out documentation for more!</a>
+
+### Colours
```

Comment:
This is a messy section and doesn't give a clear indication of what to use primarily.

My understanding is that we should rely on the same colours as Gutenberg uses in the editor area unless we do something out of the ordinary when we should rely on Muriel CSS custom properties.